components/menu/themes/spectrum-two.css

```diff
@@ -16,5 +16,6 @@
 		--spectrum-menu-item-background-color-hover: rgba(var(--spectrum-gray-1000-rgb), var(--spectrum-transparent-black-200-opacity));
 		--spectrum-menu-item-background-color-down: rgba(var(--spectrum-gray-1000-rgb), var(--spectrum-transparent-black-200-opacity));
 		--spectrum-menu-item-background-color-key-focus: rgba(var(--spectrum-gray-1000-rgb), var(--spectrum-transparent-black-200-opacity));
+		--spectrum-menu-item-corner-radius: var(--spectrum-corner-radius-100);
```

Comment:
We'll need the fallback value added to the `spectrum.css` file, otherwise the S1 version will inherit the v15 value for `--spectrum-corner-radius-100`